index.js

```
@@ -37,6 +38,9 @@ const app = express()
 app.use(compression())
 
 xapp.on("error", err => {
+  error(
+    "Could not start Metaphysics because it could not set up the xapp token, this is likely due to your `GRAVITY_*` env vars:"
```

Nice, yea silent failure to boot isnt good :(